tests/Unit/Api/CommandHandler/AddGiftCardToOrderHandlerTest.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Setono\SyliusGiftCardPlugin\Unit\Api\CommandHandler;
+
+use InvalidArgumentException;
+use PHPUnit\Framework\TestCase;
+use Prophecy\PhpUnit\ProphecyTrait;
+use Setono\SyliusGiftCardPlugin\Api\Command\AddGiftCardToOrder;
+use Setono\SyliusGiftCardPlugin\Api\CommandHandler\AddGiftCardToOrderHandler;
+use Setono\SyliusGiftCardPlugin\Applicator\GiftCardApplicatorInterface;
+use Setono\SyliusGiftCardPlugin\Model\GiftCard;
+use Setono\SyliusGiftCardPlugin\Repository\GiftCardRepositoryInterface;
+use Setono\SyliusGiftCardPlugin\Repository\OrderRepositoryInterface;
+use Tests\Setono\SyliusGiftCardPlugin\Application\Model\Order;
+
+final class AddGiftCardToOrderHandlerTest extends TestCase
+{
+    use ProphecyTrait;
+
+    /**
+     * @test
+     */
+    public function it_adds_gift_card_to_order(): void
```

Comment:
No assertion in this test

Reply:
Yes there is: `$giftCardApplicator->apply($order, $giftCard)->shouldBeCalled();`